examples/output_test/output_test.go

```diff
@@ -34,7 +34,7 @@ import (
 	"k8s.io/klog/v2/textlogger"
 )
 
-func newLogger(out io.Writer, v int, vmodule string) logr.Logger {
+func newLogger(out io.Writer, v int, _ string) logr.Logger {
```

Comment:
This is the one revive check that I do not fully agree with. It may be useful to spell out the parameter name here to make it more obvious what is being ignored. I could of course do that with an inline `/* vmodule */` comment, but that seems too verbose.

Comments?

Reply:
May be just a comment above the function with details of what is being ignored might help ? Since it is not being used by the function, the linter is not going to be happy anyway.

Reply:
Okay, I'll do that.

Reply:
Done, please take another look.

Reply:
Ack. This can be marked as resolved